rpc_jobs/rpc_aio.yml

```diff
@@ -47,8 +54,6 @@
             Cleanup,
             Destroy Slave
           UPGRADE_FROM_REF: "liberty-12.2"
```

Comment:
It makes sense to me to have UPGRADE_FROM_REF in the series.
This way you could have different values for different major upgrades between N and N+1

We could then use a similar var, LEAP_UPGRADE_FROM_REF in the series for leaping too.